Code/ModelsTrainer.py

```diff
@@ -34,12 +34,10 @@
 
         # extract voice features
         features_extractor = FeaturesExtractor()
-        silence_eliminator = SilenceEliminator()
 
         try   :
-            silence_eliminated_wave_file_path = "temp-" + os.path.basename(filepath).split('.')[0] + ".wav"
```

Comment:
Silence eliminator is an important part, and should be kept. 
Maybe it is good idea here, to define an extra input variable specifying where to use silence eliminator or not.
Like for example a `bool` called `use_silence_eliminator`